### ISSUE TITLE
Normalization for Ridge pipelines

### ISSUE THREAD

**Issue by baraline** [baraline/convst#13]
The performance on ASCF1 is very low in a cross validation setting for RDST + Ridge, behaviour changed after deprecation of the normalize argument for Ridge classifier. 
TODO : 
Check wheter or not the "with_mean" argument on StandardScaler should be step to False. Changing it to True seems to solve the issue.

**Comment by baraline** [comment]
By Sklearn documentation on the normalize argument : 
"This parameter is ignored when ``fit_intercept`` is set to False. If True, the regressors X will be normalized before regression by subtracting the mean and dividing by the l2-norm."

Actions to close : 
- Change all arguments of StandardScaler to `with_mean = True`
- Check consistency of experimental results once this is done. They are supposed to have been obtained with the use of the old sklearn interface with normalize=True.
